src/SegmentedControl/SegmentedControl.less

```diff
@@ -0,0 +1,27 @@
+@import "../fonts.less";
+
+.SegmentedControl {
+  display: inline-block;
+  margin-right: 10px;
+  .segmented_control--selectable-item {
+    background-color: #FFFFFF;
+    color: #566279;
+    font-size: 16px;
+    font-family: "Proxima Nova";
+    font-weight: 600;
+    text-align: center;
+    display: inline-block;
+    padding: 10px 20px;
+    border-left: 1px solid #E3E5EB;
+    &.selected {
+      background-color: #4274F6;
+      color: #FAFAFC;
+    }
+  }
+  border: 1px solid #E3E6EB;
+  border-radius: 3px;
+}
+
+.SegmentedControl:first-child {
```

Comment:
nit: move under `.SegmentedControl` as `&:first-child`

also, should this be under `.segmented_control--selectable-item` instead?
`:first-child` affects the component it's assigned to, not its children